scripts/spread-hunter-menu.ps1

```
@@ -1174,6 +1230,44 @@ function Clear-RuntimeState {
+        Lsh-Warn "$Name PID $TargetPid unavailable; not killed."
+        return $false
+    }
+}
+
+function Stop-ShadowSession {
+    <# Stop a menu-driven shadow session before its timebox ends: the market
+    screener, the rehearsal loop (shadow_run), the stop-loss watcher, and the
+    shadow viewer. Recorded PIDs are killed only when their start times match. #>
+    if (Test-Path $ShadowSessionFile) {
+        $s = $null
+        try { $s = Get-Content $ShadowSessionFile -Raw | ConvertFrom-Json } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)